README.md

```diff
@@ -95,13 +95,16 @@ process of the projects they lead.
 
 ## Project Licensing Requirement ##
 
-All of the projects of [@morganey-lang] community are required to
-be licensed under one of the following licenses:
+Here's a list of licenses recommended by [@morganey-lang] to use in the
+community projects:
```

Comment:
It should be either "recommended for use" or "recommended to be used".

The whole sentence can also be rewritten like that: "We recommend using one of the following licenses for [@morganey-lang] community projects". This wording makes it clear that you're recommending licenses, not the list itself.